src/components/Form/index.tsx

```diff
@@ -0,0 +1,70 @@
+import set = require('lodash/set');
+import * as React from 'react';
+import Form from 'react-jsonschema-form';
+import { FormProps } from 'rendition';
+import styled from 'styled-components';
+import { Box } from '../Grid';
+import BaseInput from './widgets/BaseInput';
+
+const widgets = {
+	BaseInput,
+};
+
+const FormWrapper = styled(Box)`
+	fieldset {
+		margin: 0;
+		padding: 0;
+		border: 0;
+	}
+`;
+
+export default class FormHOC extends React.Component<
+	FormProps,
+	{ formData: any }
+> {
+	constructor(props: FormProps) {
+		super(props);
+
+		this.state = {
+			formData: {},
+		};
+
+		this.change = this.change.bind(this);
+	}
+
+	change(target: string, value: any) {
+		console.log(target, value);
+		const { formData } = this.state;
+		set(formData, target, value);
+
+		console.log('FORMDATA', formData);
+
+		this.setState({ formData }, () => {
+			if (this.props.onChange) {
+				this.props.onChange(formData);
+			}
+		});
+	}
+
+	submit() {
+		if (this.props.onSubmit) {
+			this.props.onSubmit(this.state.formData);
+		}
+	}
+
+	render() {
+		const { schema, onChange, onSubmit, value, ...props } = this.props;
```

Comment:
I think that `onSubmit` isn't used.

Reply:
onSubmit isn't used, but its a neat way to stop the prop getting passed to `FormWrapper`